### PR TITLE
build: Bump deepvariant to 1.6.1

### PR DESCRIPTION
**Pull request by edmundmiller**
[Slack thread](https://nfcore.slack.com/archives/CJRH30T6V/p1718189925184689)

https://github.com/nf-core/modules/issues/1754 is why we're still manually bumping this.

### CODE REVIEW

**Review by maxulysse — APPROVED** [review]

File: modules/nf-core/deepvariant/main.nf
Comment (maxulysse) [inline modules/nf-core/deepvariant/main.nf:43]:
```suggestion
        --intermediate_results_dir=tmp \\
```

**Comment by fellen31** [comment]
Don't merge this yet, the version says 1.6.0.

**Comment by fellen31** [comment]
@edmundmiller So even though the code in this docker is from 1.6.1, the output will say 1.6.0. The developers don't seem very keen on building a new image themselves. Would you either be able to: 
```
# $ git clone https://github.com/google/deepvariant.git
# $ cd deepvariant
```
Manually update `ARG VERSION=1.6.0` to `ARG VERSION=1.6.1` in the Dockerfile before building (confirmed this is the only thing needed in the linked issue above)
```
# $ sudo docker build -t deepvariant .
```

Or use the docker at `docker.io/fellen31/deepvariant:1.6.1` where I have done the above?

**Comment by edmundmiller** [comment]
I think we shouldn't modify the Docker image and update the version inside for "reproducibility issues". Wouldn't want people to get upset because the hashes don't match or something 🙃.

**Comment by fellen31** [comment]
So just double checking, you are happy with having `nf-core/deepvariant:1.6.1` writing version 1.6.0 in the output VCF?

**Comment by maxulysse** [comment]
I think, it's a known issue, so I'm happy enough, that's what the tool itself report, we could have add an env and report 1.6.1 instead to be fair.
@fellen31 would you rather we do that?

**Comment by fellen31** [comment]
I’d argue that for the end user it might be more confusing to have a mismatch in versions between a multiqc report (1.6.1) and vcf file (1.6.0) using the 1.6.1 docker image (like now), than having 1.6.0 written in both multiqc and vcf, even though it’s really 1.6.1. I can always patch with my own docker, so I’ll leave it up to you or @ramprasadn. As long as you are aware if you update Sarek or raredisease.

**Comment by maxulysse** [comment]
good point, I'll create a PR then

**Comment by maxulysse** [comment]
It is already done actually: `def VERSION = '1.6.1'`, and that's what we report

**Comment by fellen31** [comment]
> It is already done actually: `def VERSION = '1.6.1'`, and that's what we report

Yea, but the VCF header would still say 1.6.0. But maybe nobody looks 😄

**Comment by maxulysse** [comment]
Oh I see.

Can we just edit the vcf.gz header and change that then?

**Comment by maxulysse** [comment]
I agree with @edmundmiller I don't want to modify the container itself

**Comment by edmundmiller** [comment]
Ah, I was afraid it might mess with the output 😞 

I think we should modify the container then. If DeepVariant is fine with confusing people so be it.

We could also just use `1.6.0` and skip `1.6.1` because they refuse to push a new container, and anyone with the sharding issue can manually update the container?

**Comment by maxulysse** [comment]
I don't really have a preference there...
We cannot fix everything at our level

**Pull request merged**
